### PR TITLE
Update custom-skins.md

### PR DESCRIPTION
**Pull request by yaichenbaum**

### CODE REVIEW

File: docs/articles/custom-skins.md [hunk 1]
```diff
@@ -28,89 +28,53 @@ Default template
...
+            <Color x:Key="SolidBackgroundFillColorSecondary">#7F3C4E</Color>
+            <Color x:Key="SolidBackgroundFillColorTertiary">#763B4A</Color>
+            <Color x:Key="SolidBackgroundFillColorQuarternary">#592A37</Color>
             <SolidColorBrush x:Key="SolidBackgroundFillColorBaseBrush" Color="{ThemeResource SolidBackgroundFillColorBase}" />
             <SolidColorBrush x:Key="SolidBackgroundFillColorSecondaryBrush" Color="{ThemeResource SolidBackgroundFillColorSecondary}" />
             <SolidColorBrush x:Key="SolidBackgroundFillColorTertiaryBrush" Color="{ThemeResource SolidBackgroundFillColorTertiary}" />
             <SolidColorBrush x:Key="SolidBackgroundFillColorQuarternaryBrush" Color="{ThemeResource SolidBackgroundFillColorQuarternary}" />
             <!-- Acrylic Resources -->
-            <Color x:Key="SolidBackgroundAcrylic">#332041</Color>
+            <Color x:Key="SolidBackgroundAcrylic">#592A37</Color>
```
Comment (chingucoding) [inline docs/articles/custom-skins.md:41]: Any specific reason for the color change here?

Comment (yaichenbaum) [reply docs/articles/custom-skins.md:41]: I just copied the values from an updated theme so that's why they changed.

Comment (chingucoding) [reply docs/articles/custom-skins.md:41]: I assume the other resources being removed is due to the same?

Comment (yaichenbaum) [reply docs/articles/custom-skins.md:41]: Correct. This is pretty much a copy of an existing theme.

Comment (chingucoding) [reply docs/articles/custom-skins.md:41]: Sorry for the many questions, but is this how users should create new themes? Some resources (such as overriding the selected tab color) are missing now. I'll approve this PR, if you are fine for this being the best way to customize this, feel free to merge this PR.

**Review by chingucoding — APPROVED** [review]
LGTM

File: docs/articles/custom-skins.md (hunk 1 above)
Comment (yaichenbaum) [reply docs/articles/custom-skins.md:41]: Tabs can still be modified, but standard themes shouldn't need to modify them.

**Pull request merged**